be1-go/channel/election/mod.go

```diff
@@ -212,47 +210,57 @@ func (c *Channel) Catchup(catchup method.Catchup) []message.Message {
 }
 
 // Broadcast is used to handle a broadcast message.
-func (c *Channel) Broadcast(broadcast method.Broadcast, socket socket.Socket) error {
+func (c *Channel) Broadcast(broadcast method.Broadcast, _ socket.Socket) error {
 	err := c.verifyMessage(broadcast.Params.Message)
 	if err != nil {
 		return xerrors.Errorf("failed to verify broadcast message on an "+
 			"election channel: %w", err)
 	}
 
-	err = c.handleMessage(broadcast.Params.Message, socket)
+	msg := broadcast.Params.Message
+	data := msg.Data
+
+	jsonData, err := base64.URLEncoding.DecodeString(data)
```

Comment:
Why adding this part ? It would seem that it is alreday done by the register.Process function called by handleMessage ?

Reply:
It is related to the next comment you made. The tests expected that receiving a broadcast of a message that isn't a election object should return no errors. Since then Noémien responded to my message on Slack and I made the change but didn't push.